.github/ISSUE_TEMPLATE/enhancement-refinement-template.md

```diff
@@ -0,0 +1,28 @@
+# Acceptance criteria
+
+## Base requirements
+
+- [ ] User stories describing what new component is meant to achieve
+- [ ] Create initial component
+- [ ] Include stories for component
+- [ ] Add test coverage, reaching our baseline 80% coverage threshold (this may
```

Comment:
Testing is often broken into a separate issue. Are we thinking to include this so we can keep track and quickly convert to a subissue once ready?

Reply:
Yep, that was my thinking. I just didn't want to not make any mention of testing since I think it's an integral part of delivering a new component, even if we break it out as a separate task.